requirements.dev.txt

```diff
@@ -1,6 +1,6 @@
-django-debug-toolbar==1.11
+django-debug-toolbar==2.2
 django-debug-toolbar-requests==1.0.5
-django-elasticsearch-debug-toolbar==1.2.0
-pylint==2.3.1
-pylint-django==2.0.11
+django-elasticsearch-debug-toolbar==2.0.0
+pylint==2.4.4
+pylint-django==3.9.0
```

Comment:
For some reason this version of `pylint-django` doesn't seem to exist 🤷‍♂️:

https://pypi.org/project/pylint-django/#history

Latest version seems to be `2.0.15` from the April 14, 2020.

Reply:
OK... we should use 2.0.15 and assume `pylint-django` have retracted/messed up a version.

Reply:
yeah that was weird, how did we ended up with a version that doesn't even exist? 🤔 (unless they unpublish it or something) - oh well, I guess it doesn't matter.